core/identity-management/iin-agent/src/protocols/localOperations.ts

```diff
@@ -14,6 +15,121 @@ export const requestAttestation = async (counterAttestedMembership: iin_agent_pb
 };
 
 // Processes attestations on a foreign security domain unit's state received from a local IIN agent
-export const sendAttestation = async (counterAttestedMembership: iin_agent_pb.CounterAttestedMembership) => {
-    console.log('sendAttestation:');
+export const sendAttestation = async (counterAttestedMembership: iin_agent_pb.CounterAttestedMembership, securityDomain: string, memberId: string) => {
+    // Assuming only one counter attestation in the list
```

Comment:
Why?

Reply:
When some local iinAgent counter attests, it sends only its attestation, so that's why assuming only one.